perf/Cargo.toml

```diff
@@ -24,6 +24,7 @@ solana-budget-program = { path = "../programs/budget", version = "=1.6.27" }
 solana-logger = { path = "../logger", version = "=1.6.27" }
 solana-measure = { path = "../measure", version = "=1.6.27" }
 solana-metrics = { path = "../metrics", version = "=1.6.27" }
+solana-vote-program = { path = "../programs/vote", version = "=1.6.27" }
```

Comment:
don't need this dependency, the vote program id is in `solana_sdk::vote::program::id`

Reply:
It is better to ref to vote id with full path.

Reply:
Ah ok that's fine then